iset.mm

```diff
@@ -151091,6 +151259,223 @@ the additive structure must be abelian (see ~ ringcom ), care must be
   $}
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Local rings
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c LRing $.
+  $( Extend class notation with class of all local rings. $)
+  clring $a class LRing $.
+
+  ${
+    $d x y r $.
+    $( A local ring is a non-zero ring where any sum which equals one has one
```

Comment:
I think the (i)set.mm spelling is "nonzero", not hyphenated, but to check.

Reply:
As much as I want to write an elaborate essay about how a little variation won't kill us, I can do "nonzero".

We currently have both but that isn't an especially strong argument as we could of course change that.

Reply:
Done.